osquery/core/tests/scope_guard_tests.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed as defined on the LICENSE file found in the
+ *  root directory of this source tree.
+ */
+
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include <boost/core/ignore_unused.hpp>
+#include <boost/filesystem.hpp>
+
+#include <osquery/logger.h>
+#include <osquery/tests/test_util.h>
+
+#include "osquery/core/scope_guard.h"
+
+namespace fs = boost::filesystem;
+
+namespace osquery {
+
+class ScopeGuardTests : public testing::Test {};
```

Comment:
If you don't need a setup for your tests you can omit creation of a fixture completely and use `TEST()` instead of `TEST_F()`

Reply:
You right, there is no big difference between `GTEST` and `TEST_F` if you don't need any state or prepared environment. But it is a bit useful to have a defined cpp name for the group of tests, which prevent you from something like:
```c++
GTEST(ScopeGuardTests, test1) {...}
GTEST(ScoupeGuardTests, test1) {...}
```
I'd like to be sure that all of tests from the same group have the same prefix. Does it make sense for you?

Reply:
Makes sense!